utils/schema/usersSchema.js

```diff
@@ -4,14 +4,55 @@ const { config } = require('../../config');
 const schema = mongoose.Schema;
 
 const UsersSchema = new schema({
-  FirstName: {
+  documentID: {
+    type: Number,
+    unique: true,
+    trim: true,
+    required: 'Please Supply an identification number',
+  },
+  firstName: {
+    type: String,
+  },
+  lastName: {
+    type: String,
+  },
+  email: {
     type: String,
+    unique: true,
+    lowercase: true,
+    trim: true,
+    required: 'Please Supply an email address',
   },
-  LastName: {
+  contactNumber: {
+    type: Number,
+  },
+  username: {
+    type: String,
+    unique: true,
+    lowercase: true,
+    trim: true,
+    default: function () {
```

Comment:
I think this should be in userServices. You can't control the uniqueness of the username here.